docs/index.rst

```diff
@@ -52,8 +52,8 @@ How to use it
     All configurable Scrapy Settings added by the Middleware.
 
 
-With the middleware, the usage of crawlera is automatic, every request will go through crawlera without nothing to worry about.
-If you want to *disable* crawlera on a specific Request, you can do so by updating `meta` with `dont_proxy=True`::
+With the middleware, the usage of Zyte Smart Proxy Manager is automatic, every request will go through Smart Proxy Manager without nothing to worry about.
+If you want to *disable* Smart Proxy Manager on a specific Request, you can do so by updating `meta` with `dont_proxy=True`::
```

Comment:
I’m relatively sure that we are supposed to keep the Zyte prefix everywhere, even within the same sentence.

```suggestion
With the middleware, the usage of Zyte Smart Proxy Manager is automatic, every request will go through Zyte Smart Proxy Manager without nothing to worry about.
If you want to *disable* Zyte Smart Proxy Manager on a specific Request, you can do so by updating `meta` with `dont_proxy=True`::
```

What we can drop after the first mention is the “ (formerly Crawlera)” suffix, which I think we should be using in the first product mention in the README and maybe also in the first mention in this main page of the documentation.